fix: SlackMessageInternalServiceTest fail

```
> Task :test

woowacrew.article.slack.service.SlackMessageInternalServiceTest > saveFailBecauseAuthorId() FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: java.lang.IllegalArgumentException

woowacrew.article.slack.service.SlackMessageInternalServiceTest > saveFailBecauseToken() FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: java.lang.IllegalArgumentException

woowacrew.article.slack.service.SlackMessageInternalServiceTest > findAll() FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: java.lang.IllegalArgumentException

woowacrew.article.slack.service.SlackMessageInternalServiceTest > findById() FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: java.lang.IllegalArgumentException

woowacrew.article.slack.service.SlackMessageInternalServiceTest > save() FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: java.lang.IllegalArgumentException

woowacrew.article.slack.service.SlackMessageInternalServiceTest > saveFailBecauseChannelId() FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: java.lang.IllegalArgumentException

woowacrew.article.slack.service.SlackMessageInternalServiceTest > findByNotExistId() FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: java.lang.IllegalArgumentException

woowacrew.article.slack.service.SlackMessageInternalServiceTest > notFoundRecentlySlackMessage() FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: java.lang.IllegalArgumentException

woowacrew.article.slack.service.SlackMessageInternalServiceTest > findRecentlyMessage() FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: java.lang.IllegalArgumentException

17:13:14.057 [INFO ] [Thread-6] [o.s.s.c.ThreadPoolTaskExecutor] - Shutting down ExecutorService 'applicationTaskExecutor'
17:13:14.060 [INFO ] [Thread-13] [o.s.s.c.ThreadPoolTaskExecutor] - Shutting down ExecutorService 'applicationTaskExecutor'
17:13:14.061 [INFO ] [Thread-11] [o.s.o.j.LocalContainerEntityManagerFactoryBean] - Closing JPA EntityManagerFactory for persistence unit 'default'
17:13:14.061 [INFO ] [Thread-11] [o.h.t.s.i.SchemaDropperImpl$DelayedDropActionImpl] - HHH000477: Starting delayed evictData of schema as part of SessionFactory shut-down'
Hibernate: drop table anonymous_article if exists
17:13:14.062 [INFO ] [Thread-15] [o.s.o.j.LocalContainerEntityManagerFactoryBean] - Closing JPA EntityManagerFactory for persistence unit 'default'
17:13:14.062 [INFO ] [Thread-15] [o.h.t.s.i.SchemaDropperImpl$DelayedDropActionImpl] - HHH000477: Starting delayed evictData of schema as part of SessionFactory shut-down'
Hibernate: drop table anonymous_article if exists
17:13:14.063 [INFO ] [Thread-43] [o.s.s.c.ThreadPoolTaskExecutor] - Shutting down ExecutorService 'applicationTaskExecutor'
17:13:14.064 [INFO ] [Thread-43] [o.s.o.j.LocalContainerEntityManagerFactoryBean] - Closing JPA EntityManagerFactory for persistence unit 'default'
Hibernate: drop table article if exists
Hibernate: drop table comment if exists
Hibernate: drop table crew_article if exists
Hibernate: drop table degree if exists
Hibernate: drop table feed_article if exists
Hibernate: drop table article if exists
Hibernate: drop table comment if exists
Hibernate: drop table crew_article if exists
Hibernate: drop table degree if exists
Hibernate: drop table feed_article if exists
Hibernate: drop table feed_source if exists
Hibernate: drop table keyword if exists
Hibernate: drop table slack_message if exists
Hibernate: drop table user if exists
Hibernate: drop table feed_source if exists
Hibernate: drop table keyword if exists
Hibernate: drop table slack_message if exists
Hibernate: drop table user if exists
17:13:14.073 [INFO ] [Thread-43] [o.h.t.s.i.SchemaDropperImpl$DelayedDropActionImpl] - HHH000477: Starting delayed evictData of schema as part of SessionFactory shut-down'
17:13:14.078 [INFO ] [Thread-40] [o.s.s.c.ThreadPoolTaskExecutor] - Shutting down ExecutorService 'applicationTaskExecutor'
17:13:14.082 [INFO ] [Thread-43] [o.s.b.f.s.DisposableBeanAdapter] - Invocation of destroy method failed on bean with name 'inMemoryDatabaseShutdownExecutor': org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
17:13:14.082 [INFO ] [Thread-43] [com.zaxxer.hikari.HikariDataSource] - HikariPool-20 - Shutdown initiated...
17:13:14.086 [INFO ] [Thread-43] [com.zaxxer.hikari.HikariDataSource] - HikariPool-20 - Shutdown completed.

203 tests completed, 9 failed

> Task :test FAILED

```

--- a/src/test/java/woowacrew/article/slack/service/SlackMessageInternalServiceTest.java
+++ b/src/test/java/woowacrew/article/slack/service/SlackMessageInternalServiceTest.java
@@ -25,9 +25,12 @@ import java.util.Optional;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
-@SpringBootTest(properties = "spring.config.location=classpath:/slack.yml",
+@SpringBootTest(properties = "spring.config.location=classpath:/slack.yml, classpath:/application.yml",
         webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 class SlackMessageInternalServiceTest {
 

